site/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+    background-color: black;}
+
+nav {text-align: center;
+		font-family: "Impact", Times, serif;
+			color: #BFD2D9;}
+
+
+
+p {font-family: "Impact", Times, serif;
+		color:#0BA5D9}
+h1 {text-align: center;
+		font-family: "Impact", Times, serif;
+			color:#0BA5D9;}
+h2 {font-family: "Impact", Times, serif;
+		color:#0BA5D9;}
+
```

Comment:
Be mindful of indentation here too. For example, in the nav selector, font-family and color should be at the same level as text-align because they are not "children".